src/toolkits/activity_classification/ac_data_iterator.cpp

```diff
@@ -132,152 +132,284 @@ variant_map_type _activity_classifier_prepare_data_impl(const gl_sframe &data,
                                                         const int &predictions_in_chunk,
                                                         const std::string &target,
                                                         bool verbose) {
+  DASSERT_TRUE(features.size() > 0);
```

Comment:
Let's just revert these changes. Don't run clang-format on this bit. In the final squashed commit, let's not touch the history for these lines, and then it will be easier to review the overall change. And we don't have to worry about the flaky unit test for the Python path.